system_test/connectionOptions.js

```diff
@@ -14,6 +14,9 @@ let snowflakeTestHost = process.env.SNOWFLAKE_TEST_HOST;
 let snowflakeTestPort = process.env.SNOWFLAKE_TEST_PORT;
 let snowflakeTestProxyHost = process.env.SNOWFLAKE_TEST_PROXY_HOST;
 let snowflakeTestProxyPort = process.env.SNOWFLAKE_TEST_PROXY_PORT;
+const snowflakeTestProxyProtocol = process.env.SNOWFLAKE_TEST_PROXY_PROTOCOL;
+const snowflakeTestProxyUser = process.env.SNOWFLAKE_TEST_PROXY_USER;
+const snowflakeTestProxyPassword = process.env.SNOWFLAKE_TEST_PROXY_PASSWORD;
```

Comment:
I added these environment variables so that you can set the `proxyProtocol`, `proxyUser`, and `proxyPassword` options for the proxy agent in the "testProxy" system test. I also ran the file through the linter to fix lint warnings.

Reply:
Ok